packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -84,6 +86,22 @@ class TooltipThemeData with Diagnosticable {
   /// The length of time that the tooltip will be shown once it has appeared.
   final Duration? showDuration;
 
+  /// The [TooltipTriggerMode] that will show the tooltip.
+  ///
+  /// This value is used if [Tooltip.triggerMode] is null.
+  /// Read [Tooltip.triggerMode] documentation for other sources.
+  final TooltipTriggerMode? triggerMode;
+
+  /// Whether the tooltip should provide acoustic and/or haptic feedback.
+  ///
+  /// For example, on Android a tap will produce a clicking sound and a
+  /// long-press will produce a short vibration, when feedback is enabled.
+  ///
```

Comment:
Also here, default behavior

Reply:
Are you sure you want it here too? `TooltipTheme` doesn't provide any default value for it.

Reply:
It might be nice to specify what the default behavior for this property in Tooltip is.

Reply:
you can just say yes xd